tests/test_reconciliation_lock_order_red.py

```
@@ -0,0 +1,185 @@
+                            (
+                                self.case.scope["tenant_account_id"],
+                                self.case.run_reference,
+                                match_id,
+                                "stmt-lock-order",
+                            ),
+                        )
+                        allocation_connection.commit()
+                    except psycopg.errors.DeadlockDetected:
+                        allocation_connection.rollback()
+                        worker_outcome.put("deadlock")
+                    except Exception as error:  # pragma: no cover - diagnostic boundary
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)